rspace/src/main/scala/coop/rchain/rspace/trace/Event.scala

```diff
@@ -45,7 +45,10 @@ final case class COMM(
     produces: Seq[Produce],
     peeks: SortedSet[Int],
     timesRepeated: Map[Produce, Int]
-) extends Event
+) extends Event {
+  def matches(datum: Datum[_], produceCounter: => Int): Boolean =
+    timesRepeated.get(datum.source).exists(cnt => datum.persist || cnt == produceCounter)
```

Comment:
Nitpick: based on what I've heard about abbreviated value names in the Haskell world, I prefer `count` to `cnt`.

Reply:
Fixed in fb0526681397ec4e067344109bb88797ea6c568d